src/m3/test/json-validator.ts

```diff
@@ -25,7 +25,7 @@ export const createJsonValidatorForSchema = (schema: unknown): (json: unknown) =
 }
 
 
-const metaSchema = JSON.parse(Deno.readTextFileSync(`${__dirname(import.meta)}/json.schema.json`))
+const metaSchema = JSON.parse(Deno.readTextFileSync(`src/m3/test/json.schema.json`))
```

Comment:
`__dirname(import.meta)` wasn't working on the Node.js-side?

Reply:
Nope. It errors with
```
Running tests in ./script/m3/test/library.test.js...

/Users/alessio/projects/lioncore-typescript/npm/script/m3/test/json-validator.js:50
const metaSchema = JSON.parse(dntShim.Deno.readTextFileSync(`${(0, local_files_js_1.__dirname)(import.meta)}/json.schema.json`));
                                                                                                      ^^^^

SyntaxError: Cannot use 'import.meta' outside a module
    at Object.compileFunction (node:vm:352:18)
    at wrapSafe (node:internal/modules/cjs/loader:1031:15)
    at Module._compile (node:internal/modules/cjs/loader:1065:27)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
```

etc. etc.